facebook-commerce.php

```diff
@@ -2438,41 +2438,6 @@ public function init_form_fields() {
 
 		$form_fields = [
 
-			/** @see \WC_Facebookcommerce_Integration::generate_manage_connection_title_html() */
-			[
-				'type'  => 'manage_connection_title',
-			],
-
-			/** @see \WC_Facebookcommerce_Integration::generate_facebook_page_name_html() */
-			self::SETTING_FACEBOOK_PAGE_ID => [
-				'type'    => 'facebook_page_name',
-				'default' => '',
-			],
-
-			/** @see \WC_Facebookcommerce_Integration::generate_facebook_pixel_id_html() */
-			self::SETTING_FACEBOOK_PIXEL_ID => [
-				'type'    => 'facebook_pixel_id',
-				'default' => '',
-			],
-
-			self::SETTING_ENABLE_ADVANCED_MATCHING => [
-				'title'       => __( 'Use Advanced Matching', 'facebook-for-woocommerce' ),
-				'description' => sprintf(
-					/* translators: Placeholders: %1$s - opening <a> HTML link tag, %2$s - closing </a> HTML link tag */
-					__( 'Improve the ability to match site visitors to people on Facebook by passing additional site visitor information (such as email address or phone number). %1$sLearn more%2$s.', 'facebook-for-woocommerce' ),
-					'<a href="https://developers.facebook.com/docs/facebook-pixel/advanced/advanced-matching" target="_blank">',
-					'</a>'
-				),
-				'type'        => 'checkbox',
-				'label'       => ' ',
-				'default'     => 'yes',
-			],
-
-			/** @see \WC_Facebookcommerce_Integration::generate_create_ad_html() */
-			[
-				'type'  => 'create_ad',
-			],
-
 			[
```

Comment:
Should we maybe add a TODO here to remember to remove this later?

Reply:
b73e970e72780a4b49842ad24f6f2f62898276bf